diff --git a/Mk/Uses/pyqt.mk b/Mk/Uses/pyqt.mk
index 54fff0299196..c7a3b9d932b6 100644
--- a/Mk/Uses/pyqt.mk
+++ b/Mk/Uses/pyqt.mk
@@ -110,10 +110,10 @@ MASTER_SITES_PYQTWEBENGINE=	${MASTER_SITES_PYQT${_PYQT_VERSION}WEBENGINE}
 # Qt version-agnostic components
 PYQTBUILDER_VERSION=		1.19.0
 QSCI2_VERSION=			2.14.1
-SIP_VERSION=			6.13.1	# ,1
+SIP_VERSION=			6.14.0	# ,1
 
 # Qt 5 components
-PYQT5SIP_VERSION=		12.17.0
+PYQT5SIP_VERSION=		12.17.1
 PYQT5_VERSION=			5.15.11
 PYQT53D_VERSION=		5.15.7
 PYQT5CHARTS_VERSION=		5.15.7
@@ -123,13 +123,13 @@ PYQT5WEBENGINE_VERSION=		5.15.7
 
 # Qt 6 components
 PYQT6SIP_VERSION=		13.10.2
-PYQT6_VERSION=			6.9.1
-PYQT63D_VERSION=		6.9.0
-PYQT6CHARTS_VERSION=		6.9.0
-PYQT6DATAVIS3D_VERSION=		6.9.0
-PYQT6GRAPHS_VERSION=		6.9.0
-PYQT6NETWORKAUTH_VERSION=	6.9.0
-PYQT6WEBENGINE_VERSION=		6.9.0
+PYQT6_VERSION=			6.10.0
+PYQT63D_VERSION=		6.10.0
+PYQT6CHARTS_VERSION=		6.10.0
+PYQT6DATAVIS3D_VERSION=		6.10.0
+PYQT6GRAPHS_VERSION=		6.10.0
+PYQT6NETWORKAUTH_VERSION=	6.10.0
+PYQT6WEBENGINE_VERSION=		6.10.0
 
 # Qt version-agnostic components
 PYQTBUILDER_DISTNAME=		pyqt_builder-${PYQTBUILDER_VERSION}
diff --git a/devel/py-qt5-sip/distinfo b/devel/py-qt5-sip/distinfo
index e9a3c08b9c85..30d660bd63eb 100644
--- a/devel/py-qt5-sip/distinfo
+++ b/devel/py-qt5-sip/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1738581039
-SHA256 (pyqt5_sip-12.17.0.tar.gz) = 682dadcdbd2239af9fdc0c0628e2776b820e128bec88b49b8d692fe682f90b4f
-SIZE (pyqt5_sip-12.17.0.tar.gz) = 104042
+TIMESTAMP = 1761268987
+SHA256 (pyqt5_sip-12.17.1.tar.gz) = 0eab72bcb628f1926bf5b9ac51259d4fa18e8b2a81d199071135458f7d087ea8
+SIZE (pyqt5_sip-12.17.1.tar.gz) = 104136
diff --git a/devel/py-qt5-sip/files/patch-pyproject.toml b/devel/py-qt5-sip/files/patch-pyproject.toml
index e72a7c13302f..8a16e2b4ac86 100644
--- a/devel/py-qt5-sip/files/patch-pyproject.toml
+++ b/devel/py-qt5-sip/files/patch-pyproject.toml
@@ -3,9 +3,9 @@ with 63.x.
 
 [1] https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=270358
 
---- pyproject.toml.orig	2024-12-12 10:01:35 UTC
+--- pyproject.toml.orig	2025-10-08 08:47:54 UTC
 +++ pyproject.toml
 @@ -1,2 +1,2 @@
  [build-system]
--requires = ["setuptools >=69.5"]
+-requires = ["setuptools >=75.8.1"]
 +requires = ["setuptools >=63"]
diff --git a/devel/py-qt6-pyqt/distinfo b/devel/py-qt6-pyqt/distinfo
index 54f3b7ad4996..ab09537c6a34 100644
--- a/devel/py-qt6-pyqt/distinfo
+++ b/devel/py-qt6-pyqt/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1749201625
-SHA256 (pyqt6-6.9.1.tar.gz) = 50642be03fb40f1c2111a09a1f5a0f79813e039c15e78267e6faaf8a96c1c3a6
-SIZE (pyqt6-6.9.1.tar.gz) = 1067230
+TIMESTAMP = 1761268973
+SHA256 (pyqt6-6.10.0.tar.gz) = 710ecfd720d9a03b2c684881ae37f528e11d17e8f1bf96431d00a6a73f308e36
+SIZE (pyqt6-6.10.0.tar.gz) = 1079921
diff --git a/devel/py-qt6-pyqt/pkg-plist b/devel/py-qt6-pyqt/pkg-plist
index 913c9adb486d..87d2f3bbefc3 100644
--- a/devel/py-qt6-pyqt/pkg-plist
+++ b/devel/py-qt6-pyqt/pkg-plist
@@ -149,6 +149,7 @@ bin/pyuic6
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qpoint.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qprocess.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qpropertyanimation.sip
+%%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qpycore_native_interface.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qpycore_qhash.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qpycore_qlist.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qpycore_qmap.sip
@@ -158,6 +159,7 @@ bin/pyuic6
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qpycore_std_pair.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qpycore_virtual_error_handler.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qrandom.sip
+%%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qrangemodel.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qreadwritelock.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qrect.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtCore/qregularexpression.sip
@@ -255,6 +257,7 @@ bin/pyuic6
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/opengl_types.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qabstractfileiconprovider.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qabstracttextdocumentlayout.sip
+%%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qaccessibilityhints.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qaction.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qactiongroup.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qbackingstore.sip
@@ -311,6 +314,7 @@ bin/pyuic6
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qpixmapcache.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qpointingdevice.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qpolygon.sip
+%%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qpygui_native_interface.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qpygui_qlist.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qpygui_vulkan.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtGui/qquaternion.sip
@@ -387,6 +391,7 @@ bin/pyuic6
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtMultimedia/qmediaplayer.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtMultimedia/qmediarecorder.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtMultimedia/qmediatimerange.sip
+%%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtMultimedia/qplaybackoptions.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtMultimedia/qpymultimedia_qlist.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtMultimedia/qscreencapture.sip
 %%PYTHON_SITELIBDIR%%/PyQt6/bindings/QtMultimedia/qsoundeffect.sip
diff --git a/graphics/py-qt6-3d/Makefile b/graphics/py-qt6-3d/Makefile
index c9c55e4b0e89..7e6e476a7f01 100644
--- a/graphics/py-qt6-3d/Makefile
+++ b/graphics/py-qt6-3d/Makefile
@@ -1,6 +1,5 @@
 PORTNAME=	3d
 DISTVERSION=	${PYQT3D_VERSION}
-PORTREVISION=	1
 CATEGORIES=	graphics python
 MASTER_SITES=	${MASTER_SITES_PYQT3D}
 PKGNAMEPREFIX=	${PYQT_PY_RELNAME}-
diff --git a/graphics/py-qt6-3d/distinfo b/graphics/py-qt6-3d/distinfo
index bacf12192d09..f891b6083de6 100644
--- a/graphics/py-qt6-3d/distinfo
+++ b/graphics/py-qt6-3d/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1743732519
-SHA256 (pyqt6_3d-6.9.0.tar.gz) = af4b497e34f30e8dba53da2f2683e82994bc6d6f512fb7a91c3150aa31b6d49a
-SIZE (pyqt6_3d-6.9.0.tar.gz) = 7469403
+TIMESTAMP = 1761269065
+SHA256 (pyqt6_3d-6.10.0.tar.gz) = 93d89fe30d98804b0983e9b52079d15ae82b1f0a279a5f90f2ff48ed3e6489ed
+SIZE (pyqt6_3d-6.10.0.tar.gz) = 7471242
diff --git a/graphics/py-qt6-3d/pkg-plist b/graphics/py-qt6-3d/pkg-plist
index 0b8dd9df851b..09bc67ebfabf 100644
--- a/graphics/py-qt6-3d/pkg-plist
+++ b/graphics/py-qt6-3d/pkg-plist
@@ -236,3 +236,4 @@
 %%PYTHON_SITELIBDIR%%/pyqt6_3d-%%PYQT3D_VERSION%%.dist-info/INSTALLER
 %%PYTHON_SITELIBDIR%%/pyqt6_3d-%%PYQT3D_VERSION%%.dist-info/METADATA
 %%PYTHON_SITELIBDIR%%/pyqt6_3d-%%PYQT3D_VERSION%%.dist-info/RECORD
+%%PYTHON_SITELIBDIR%%/pyqt6_3d-%%PYQT3D_VERSION%%.dist-info/licenses/LICENSE
diff --git a/net/py-qt6-networkauth/Makefile b/net/py-qt6-networkauth/Makefile
index a12de94a4bde..7a486a550fce 100644
--- a/net/py-qt6-networkauth/Makefile
+++ b/net/py-qt6-networkauth/Makefile
@@ -1,6 +1,5 @@
 PORTNAME=	networkauth
 DISTVERSION=	${PYQTNETWORKAUTH_VERSION}
-PORTREVISION=	1
 CATEGORIES=	net devel python
 MASTER_SITES=	${MASTER_SITES_PYQTNETWORKAUTH}
 PKGNAMEPREFIX=	${PYQT_PY_RELNAME}-
diff --git a/net/py-qt6-networkauth/distinfo b/net/py-qt6-networkauth/distinfo
index 528e5d14955c..095986c5166d 100644
--- a/net/py-qt6-networkauth/distinfo
+++ b/net/py-qt6-networkauth/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1743732662
-SHA256 (pyqt6_networkauth-6.9.0.tar.gz) = 9acb6e97bd54584bbaeac2aabc40ec17a79d868f7da37a163c7bd4b7a8f04b09
-SIZE (pyqt6_networkauth-6.9.0.tar.gz) = 21591
+TIMESTAMP = 1761269265
+SHA256 (pyqt6_networkauth-6.10.0.tar.gz) = 94c9504613c8ff68f08eb1ff6ba7804c277b56e335baa6e44c1eba5279961f7b
+SIZE (pyqt6_networkauth-6.10.0.tar.gz) = 23658
diff --git a/net/py-qt6-networkauth/pkg-plist b/net/py-qt6-networkauth/pkg-plist
index b7e7bd79b5f6..9ed021483d8f 100644
--- a/net/py-qt6-networkauth/pkg-plist
+++ b/net/py-qt6-networkauth/pkg-plist
@@ -15,3 +15,4 @@
 %%PYTHON_SITELIBDIR%%/pyqt6_networkauth-%%PYQTNETWORKAUTH_VERSION%%.dist-info/INSTALLER
 %%PYTHON_SITELIBDIR%%/pyqt6_networkauth-%%PYQTNETWORKAUTH_VERSION%%.dist-info/METADATA
 %%PYTHON_SITELIBDIR%%/pyqt6_networkauth-%%PYQTNETWORKAUTH_VERSION%%.dist-info/RECORD
+%%PYTHON_SITELIBDIR%%/pyqt6_networkauth-%%PYQTNETWORKAUTH_VERSION%%.dist-info/licenses/LICENSE
diff --git a/www/py-qt6-webengine/Makefile b/www/py-qt6-webengine/Makefile
index 81ae168450ea..becd950af8ac 100644
--- a/www/py-qt6-webengine/Makefile
+++ b/www/py-qt6-webengine/Makefile
@@ -1,6 +1,5 @@
 PORTNAME=	webengine
 DISTVERSION=	${PYQTWEBENGINE_VERSION}
-PORTREVISION=	1
 CATEGORIES=	www devel python
 MASTER_SITES=	${MASTER_SITES_PYQTWEBENGINE}
 PKGNAMEPREFIX=	${PYQT_PY_RELNAME}-
diff --git a/www/py-qt6-webengine/distinfo b/www/py-qt6-webengine/distinfo
index 1d8386e1f8ae..e43491e531a6 100644
--- a/www/py-qt6-webengine/distinfo
+++ b/www/py-qt6-webengine/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1743732713
-SHA256 (pyqt6_webengine-6.9.0.tar.gz) = 6ae537e3bbda06b8e06535e4852297e0bc3b00543c47929541fcc9b11981aa25
-SIZE (pyqt6_webengine-6.9.0.tar.gz) = 34616
+TIMESTAMP = 1761269283
+SHA256 (pyqt6_webengine-6.10.0.tar.gz) = 267d27275d0c79ae270bca4b03520a41fa7e85c2a4d9632da8cb9cc233a55ad1
+SIZE (pyqt6_webengine-6.10.0.tar.gz) = 37053
diff --git a/www/py-qt6-webengine/pkg-plist b/www/py-qt6-webengine/pkg-plist
index 2c7525047edf..3ab3752f93b2 100644
--- a/www/py-qt6-webengine/pkg-plist
+++ b/www/py-qt6-webengine/pkg-plist
@@ -10,6 +10,8 @@
 %%PYQT_SIPDIR%%/QtWebEngineCore/qwebenginecookiestore.sip
 %%PYQT_SIPDIR%%/QtWebEngineCore/qwebenginedesktopmediarequest.sip
 %%PYQT_SIPDIR%%/QtWebEngineCore/qwebenginedownloadrequest.sip
+%%PYQT_SIPDIR%%/QtWebEngineCore/qwebengineextensioninfo.sip
+%%PYQT_SIPDIR%%/QtWebEngineCore/qwebengineextensionmanager.sip
 %%PYQT_SIPDIR%%/QtWebEngineCore/qwebenginefilesystemaccessrequest.sip
 %%PYQT_SIPDIR%%/QtWebEngineCore/qwebenginefindtextresult.sip
 %%PYQT_SIPDIR%%/QtWebEngineCore/qwebengineframe.sip
@@ -49,3 +51,4 @@
 %%PYTHON_SITELIBDIR%%/pyqt6_webengine-%%PYQT_WEBENGINE_VERSION%%.dist-info/INSTALLER
 %%PYTHON_SITELIBDIR%%/pyqt6_webengine-%%PYQT_WEBENGINE_VERSION%%.dist-info/METADATA
 %%PYTHON_SITELIBDIR%%/pyqt6_webengine-%%PYQT_WEBENGINE_VERSION%%.dist-info/RECORD
+%%PYTHON_SITELIBDIR%%/pyqt6_webengine-%%PYQT_WEBENGINE_VERSION%%.dist-info/licenses/LICENSE
diff --git a/x11-toolkits/py-qt6-charts/Makefile b/x11-toolkits/py-qt6-charts/Makefile
index 797592042a96..3af57b5772c8 100644
--- a/x11-toolkits/py-qt6-charts/Makefile
+++ b/x11-toolkits/py-qt6-charts/Makefile
@@ -1,6 +1,5 @@
 PORTNAME=	charts
 DISTVERSION=	${PYQTCHARTS_VERSION}
-PORTREVISION=	1
 CATEGORIES=	x11-toolkits devel python
 MASTER_SITES=	${MASTER_SITES_PYQTCHARTS}
 PKGNAMEPREFIX=	${PYQT_PY_RELNAME}-
diff --git a/x11-toolkits/py-qt6-charts/distinfo b/x11-toolkits/py-qt6-charts/distinfo
index 9da1224d89c1..208027c1d6d9 100644
--- a/x11-toolkits/py-qt6-charts/distinfo
+++ b/x11-toolkits/py-qt6-charts/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1743732573
-SHA256 (pyqt6_charts-6.9.0.tar.gz) = 7efbe9bb7e6ad4f9845211a0efe0f91ca5e14f9362ed1ba84d55f2b8515091f7
-SIZE (pyqt6_charts-6.9.0.tar.gz) = 41661
+TIMESTAMP = 1761269193
+SHA256 (pyqt6_charts-6.10.0.tar.gz) = 91e15e28d011caa4c83881a90687b35e3d05ef57290cdd9760824c95bdac6a3e
+SIZE (pyqt6_charts-6.10.0.tar.gz) = 43337
diff --git a/x11-toolkits/py-qt6-charts/pkg-plist b/x11-toolkits/py-qt6-charts/pkg-plist
index ae4092b3bfc4..f187b2d76cf2 100644
--- a/x11-toolkits/py-qt6-charts/pkg-plist
+++ b/x11-toolkits/py-qt6-charts/pkg-plist
@@ -55,3 +55,4 @@
 %%PYTHON_SITELIBDIR%%/pyqt6_charts-%%PYQTCHARTS_VERSION%%.dist-info/INSTALLER
 %%PYTHON_SITELIBDIR%%/pyqt6_charts-%%PYQTCHARTS_VERSION%%.dist-info/METADATA
 %%PYTHON_SITELIBDIR%%/pyqt6_charts-%%PYQTCHARTS_VERSION%%.dist-info/RECORD
+%%PYTHON_SITELIBDIR%%/pyqt6_charts-%%PYQTCHARTS_VERSION%%.dist-info/licenses/LICENSE
diff --git a/x11-toolkits/py-qt6-datavis3d/Makefile b/x11-toolkits/py-qt6-datavis3d/Makefile
index 28aad958c471..0644f2794906 100644
--- a/x11-toolkits/py-qt6-datavis3d/Makefile
+++ b/x11-toolkits/py-qt6-datavis3d/Makefile
@@ -1,6 +1,5 @@
 PORTNAME=	datavis3d
 DISTVERSION=	${PYQTDATAVIS3D_VERSION}
-PORTREVISION=	1
 CATEGORIES=	x11-toolkits devel python
 MASTER_SITES=	${MASTER_SITES_PYQTDATAVIS3D}
 PKGNAMEPREFIX=	${PYQT_PY_RELNAME}-
diff --git a/x11-toolkits/py-qt6-datavis3d/distinfo b/x11-toolkits/py-qt6-datavis3d/distinfo
index ce2d24bbe3e0..6c2c13cd8a1d 100644
--- a/x11-toolkits/py-qt6-datavis3d/distinfo
+++ b/x11-toolkits/py-qt6-datavis3d/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1743732588
-SHA256 (pyqt6_datavisualization-6.9.0.tar.gz) = 1515475f1b2c37275ecf6ac74017a64fae8335d97b87fbbaf14bac3f82cdaa0a
-SIZE (pyqt6_datavisualization-6.9.0.tar.gz) = 210264
+TIMESTAMP = 1761269211
+SHA256 (pyqt6_datavisualization-6.10.0.tar.gz) = 4581c6f6f5e84f6431b01f563ef7b5036204a5f8823b0ea1ce5a083a880c4ee5
+SIZE (pyqt6_datavisualization-6.10.0.tar.gz) = 211447
diff --git a/x11-toolkits/py-qt6-datavis3d/pkg-plist b/x11-toolkits/py-qt6-datavis3d/pkg-plist
index 5acf2665bd5a..bd7449c73265 100644
--- a/x11-toolkits/py-qt6-datavis3d/pkg-plist
+++ b/x11-toolkits/py-qt6-datavis3d/pkg-plist
@@ -42,3 +42,4 @@
 %%PYTHON_SITELIBDIR%%/pyqt6_datavisualization-%%PYQTDATAVIS3D_VERSION%%.dist-info/INSTALLER
 %%PYTHON_SITELIBDIR%%/pyqt6_datavisualization-%%PYQTDATAVIS3D_VERSION%%.dist-info/METADATA
 %%PYTHON_SITELIBDIR%%/pyqt6_datavisualization-%%PYQTDATAVIS3D_VERSION%%.dist-info/RECORD
+%%PYTHON_SITELIBDIR%%/pyqt6_datavisualization-%%PYQTDATAVIS3D_VERSION%%.dist-info/licenses/LICENSE
diff --git a/x11-toolkits/py-qt6-graphs/Makefile b/x11-toolkits/py-qt6-graphs/Makefile
index d03d1fe5a821..8b06b4ef243b 100644
--- a/x11-toolkits/py-qt6-graphs/Makefile
+++ b/x11-toolkits/py-qt6-graphs/Makefile
@@ -1,6 +1,5 @@
 PORTNAME=	graphs
 DISTVERSION=	${PYQTGRAPHS_VERSION}
-PORTREVISION=	1
 CATEGORIES=	x11-toolkits devel python
 MASTER_SITES=	${MASTER_SITES_PYQTGRAPHS}
 PKGNAMEPREFIX=	${PYQT_PY_RELNAME}-
diff --git a/x11-toolkits/py-qt6-graphs/distinfo b/x11-toolkits/py-qt6-graphs/distinfo
index f05456eadf08..b24ce89254a5 100644
--- a/x11-toolkits/py-qt6-graphs/distinfo
+++ b/x11-toolkits/py-qt6-graphs/distinfo
@@ -1,3 +1,3 @@
-TIMESTAMP = 1743732609
-SHA256 (pyqt6_graphs-6.9.0.tar.gz) = f96eca719ce8c8c1647695136e69837d05fe56c31ca66533db568e108c8cc919
-SIZE (pyqt6_graphs-6.9.0.tar.gz) = 35234
+TIMESTAMP = 1761269239
+SHA256 (pyqt6_graphs-6.10.0.tar.gz) = 8f84734703baf8ef6f2e17052bc2622d041d569ec138b6b87d29f57edcf2fcba
+SIZE (pyqt6_graphs-6.10.0.tar.gz) = 36658
diff --git a/x11-toolkits/py-qt6-graphs/pkg-plist b/x11-toolkits/py-qt6-graphs/pkg-plist
index f205b2f94fb0..93f63d728154 100644
--- a/x11-toolkits/py-qt6-graphs/pkg-plist
+++ b/x11-toolkits/py-qt6-graphs/pkg-plist
@@ -58,3 +58,4 @@
 %%PYTHON_SITELIBDIR%%/pyqt6_graphs-%%PYQTGRAPHS_VERSION%%.dist-info/INSTALLER
 %%PYTHON_SITELIBDIR%%/pyqt6_graphs-%%PYQTGRAPHS_VERSION%%.dist-info/METADATA
 %%PYTHON_SITELIBDIR%%/pyqt6_graphs-%%PYQTGRAPHS_VERSION%%.dist-info/RECORD
+%%PYTHON_SITELIBDIR%%/pyqt6_graphs-%%PYQTGRAPHS_VERSION%%.dist-info/licenses/LICENSE
